src/main/java/org/generationcp/middleware/dao/dms/EnvironmentDao.java

```diff
@@ -35,93 +28,61 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
-/**
- * DAO class for {@link Geolocation}.
- */
-public class GeolocationDao extends GenericDAO<Geolocation, Integer> {
-
-	private static final Logger LOG = LoggerFactory.getLogger(GeolocationDao.class);
+public class EnvironmentDao extends GenericDAO<ExperimentModel, Integer> {
```

Comment:
@darla-leafnode it means that we will have 2 different DAO's to refer to the same entity? if that's the case, then we should move the functions from ExperimentDao that refers to environments.